helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/model/ClusterModelTestHelper.java

```diff
@@ -0,0 +1,50 @@
+package org.apache.helix.controller.rebalancer.waged.model;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.apache.helix.controller.dataproviders.ResourceControllerDataProvider;
+
+public class ClusterModelTestHelper extends AbstractTestClusterModel {
```

Comment:
Just call it RebalanceAlgorithmTestHelper?

Reply:
It should not only help rebalance algorithm, any places that require instances of ClusterModels can use the helper method.

Reply:
@i3wangyi 
AbstractTestClusterModel sounds like it's actually a data model that could be used in the core algorithm. Is it possible to rename it to something like **AbstractTestClusterBase**? That's the naming convention we use for a lot of test bases. This will help avoid confusion.

Reply:
@narendly When that class was done, it is for model test only. But I agree the naming is not clear. I will think about the name later with more test cases.